### PR TITLE
chore: update submodules

### PR DESCRIPTION
**Pull request by jm-clius**
This PR updates submodules.

The main purpose is to include [a high-priority fix in nim-websock](https://github.com/status-im/nim-websock/pull/109) that causes slow performance on the prod fleet: https://github.com/status-im/infra-nim-waku/issues/50.

Although `nim-websock` is the target, we update all submodules in tandem to avoid compatibility issues between submodules.

### CODE REVIEW

**Comment by Menduist** [comment]
The CI of libp2p is failing with the new version of nim-websock, need to investigate..

**Comment by Menduist** [comment]
Green in libp2p

File: tests/v2/test_waku_keepalive.nim [hunk 1]
```diff
@@ -48,4 +48,6 @@ procSuite "Waku Keepalive":
     check:
       (await completionFut.withTimeout(5.seconds)) == true
 
-    await allFutures([node1.stop(), node2.stop()])
+    # await allFutures([node1.stop(), node2.stop()])
```
Comment (LNSD) [inline tests/v2/test_waku_keepalive.nim:51]: Is there any reason for keeping this code commented? Just asking.

**Review by LNSD — APPROVED** [review]
Just a little comment about "commented but not used code".

> From my experience, if you want actually to keep commented code, add a comment explaining why that commented code should not be deleted. Otherwise, just remove it (you can always recover it from git history).

The rest, LGTM ✅

File: tests/v2/test_waku_keepalive.nim (hunk 1 above)
Comment (jm-clius) [reply tests/v2/test_waku_keepalive.nim:51]: Good question. No, I commented it out as an experiment and forgot to remove when the experiment worked. :) Will address subsequently, as I don't want the CI chain to kick off again before I can merge. :)

Comment (jm-clius) [reply tests/v2/test_waku_keepalive.nim:51]: And you're right - we shouldn't have dead code and legacy comments cluttering the code.

Comment (LNSD) [reply tests/v2/test_waku_keepalive.nim:51]: It happens to me often, too 😆

**Review by kaiserd — APPROVED** [review]
LGTM!

**Pull request merged**